auth/user.go

```
@@ -471,6 +471,10 @@ func (user *userImpl) GetRoles() []Role {
 	return user.roles
 }
 
+func (user *userImpl) PreloadRoles() {
+	_ = user.GetRoles()
```

Why is it safe to ignore error here?

---

That returns the roles that the user has which is not needed for preloading the roles as it is added on `userImpl` in GetRoles()